app/views/home/donate.html.erb

```diff
@@ -1,12 +1,10 @@
-﻿<h2 class="heading"><%= ts('Donations') %></h2>
-<h3 class="heading"><%= ts('There are two main ways to support the AO3 - donating your time or money.') %></h3>
+﻿<h2 class="heading"><%= t(".general.title") %></h2>
+<h3 class="heading"><%= t(".general.text") %></h3>
 
 <div class="userstuff">
-  <h3><%= ts('Donating Your Time') %></h3>
-  <p><%= ts("The <a href=\"http://www.transformativeworks.org\">Organization for Transformative Works</a> is the parent organization of the Archive of Our Own. We are often looking for volunteers to contribute to <a href=\"http://www.transformativeworks.org/our-projects\">our projects</a>. If you're interested in volunteering specifically to help the Archive of Our Own, the roles to look out for include tag wranglers, testers, coders, Support staff and Abuse staff. We encourage you to browse through the <a href=\"http://www.transformativeworks.org/volunteer\">available listings</a> and apply for any roles which match your qualifications and interests.").html_safe %></p>
+  <h3><%= t(".time.title") %></h3>
+  <p><%= t(".time.text_html") %></p>
```

Comment:
Could you use variable interpolation for the link text and URLs, please? (The [i18n guidelines](https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards) have a bullet point about how we do links.)

Reply:
I thought this was only relevant for internal URL?